gem/templates/base/profiles/register.html

```diff
@@ -63,6 +63,24 @@ <h3>{% trans "Sign up" %}</h3>
           </fieldset>
         {% endif %}
 
+        {% if settings.profiles.UserProfilesSettings.show_mobile_number_field and settings.profiles.UserProfilesSettings.country_code %}
+          <fieldset>
+            <label for="mobile_number">{% trans "Enter your mobile number" %}</label>
+            {{ form.mobile_number.errors}}
+            {{ form.mobile_number }}
+            <p class="helptext">{% trans "(e.g. 0900000013)" %}</p>
+          </fieldset>
+        {% endif %}
+
+        {% if settings.profiles.UserProfilesSettings.show_email_field %}
+          <fieldset>
+            <label for="id_email">{% trans "Enter your email address" %}</label>
+            {{ form.email.errors}}
+            {{ form.email }}
+            <p class="helptext">{% trans "(e.g. example@foo.com)" %}</p>
```

Comment:
why don't we set set the help test in the field model?

Reply:
We need to select and style help-text separately here.